minimumSizeSubarraySum.py

```diff
@@ -0,0 +1,23 @@
+class Solution(object):
```

Comment:
Without any comments, the code is really hard to understand.
Please add comments.